audiomate/corpus/io/zamia_speech.py

```diff
@@ -0,0 +1,331 @@
+import os
+import re
+import shutil
+import tarfile
+
+import audiomate
+import requests
+from audiomate import annotations
+from audiomate import issuers
+from audiomate import logutil
+from audiomate.utils import download
+from audiomate.utils import textfile
+
+from . import base
+
+logger = logutil.getLogger()
+
+# ==================================================================================================
+
+urls_speech_dir = {
+    "de": "https://goofy.zamia.org/zamia-speech/corpora/zamia_de/",
+    "en": "https://goofy.zamia.org/zamia-speech/corpora/zamia_en/",
+}
+
+urls_transcripts = {
+    "de": "https://raw.githubusercontent.com/gooofy/zamia-speech/master/data/src/speech/zamia_de/transcripts_00.csv",  # noqa
+    "en": "https://raw.githubusercontent.com/gooofy/zamia-speech/master/data/src/speech/zamia_en/transcripts_00.csv",  # noqa
+}
+
+
+# ==================================================================================================
+
+
+class ZamiaSpeechDownloader(base.CorpusDownloader):
+    """ Most of the code is take from the Voxforge downloader as the structure is similar """
```

Comment:
`Most of the code is taken from the Voxforge downloader as the structure is similar.`